### ISSUE TITLE
Lost's Mantle unlock change

### ISSUE THREAD

**Issue by Xaitat** [UnderscoreKilburn/repentance-issues#1298]
I think it would be a good idea to make 879 coins in the Greed Machine unlock the starting ED6 for The Lost instead of the built-in Holy Mantle, which should be given by default. The Holy Mantle is a basic gimmick of the character, he doesn't even "hold" it anymore, yet you still need a difficult and unrelated unlock to get it. Eternal D6 is an improvement to the character, just like the other greed more unlocks, and not a "you can now actually play  the character". One of the points of Tlost is exaclty not having the mantle, yet you can technically(altough difficult) unlock him without having unlocked the mantle for the normal lost

**Comment by BadsyEdgecat** [comment]
i agree, and not only that, but doing this would also mean that downpour 2 isnt really difficult to do without greed mode completed
